controllers/kafkatopic_controller.go

```diff
@@ -107,6 +131,12 @@ func (r *KafkaTopicReconciler) Reconcile(ctx context.Context, request reconcile.
 		return r.checkFinalizers(ctx, broker, instance)
 	}
 
+	// No need to do anything when the kafka topic is not managed by Koperator
+	if !isTopicManagedByKoperator(instance) {
+		reqLogger.Info(fmt.Sprintf("topic '%s' is not managed by %s it is managed by '%s' ==> nothing to reconcile here", instance.Spec.Name, webhooks.TopicManagedByKoperatorAnnotationValue, instance.GetAnnotations()[webhooks.TopicManagedByAnnotationKey]))
```

Comment:
```suggestion
		reqLogger.Info(fmt.Sprintf("topic '%s' is not managed by %s, it is managed by '%s' ==> nothing to reconcile here", instance.Spec.Name, webhooks.TopicManagedByKoperatorAnnotationValue, managedByStatus))
```

Is there a particular reason to not use the `managedByStatus` here?

Reply:
We can use both. Same result. But I think better to use your suggestion. Thanks!